.github/workflows/superset-e2e.yml

```diff
@@ -0,0 +1,66 @@
+name: E2E
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  cypress:
+    name: Cypress
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        browser: ['chrome']
+    env:
+      SUPERSET_CONFIG: tests.superset_test_config
+      PYTHONPATH: ${{ github.workspace }}
+      REDIS_PORT: 16379
+      CI: github-actions
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+    services:
```

Comment:
We need a SQL server service in here (postgres?). SQLlite is being deprecated currently.

Reply:
Should I add a section for unit tests in SQLite?

Reply:
We are going to remove it entirely. For now, yes, we should run unit tests in SQLite as the work to deprecate hasn't been started.